tf/net.py

```diff
@@ -7,8 +7,8 @@
 import proto.net_pb2 as pb
 
 LC0_MAJOR = 0
-LC0_MINOR = 15
-LC0_PATCH = 0
+LC0_MINOR = 14
```

Comment:
I don't think we should expect LC0 version numbers to go backwards - if they have gone backwards, we should bump it ASAP.